android/build.gradle

```diff
@@ -1,21 +1,42 @@
 buildscript {
     repositories {
         jcenter()
+        maven {
+            url 'https://maven.google.com/'
```

Comment:
isn't this the same as `google()`? or is this just for compatibility?

Reply:
yeah, for compatibility https://stackoverflow.com/questions/46467561/difference-between-google-and-maven-url-https-maven-google-com